e2e/tests/helpers.bash

```diff
@@ -223,7 +228,26 @@ run_with_retry() {
       if [[ ${output} =~ ${2} ]]; then
           return 0
       fi
-      sleep ${4}
+      sleep ${wait_time}
+  done
+  echo ${1}
+  echo $output
+  return 1
+}
+
+# Run an arbitrary bash command until it succeeds
+#
+# usage: run_with_success <bash command> [retries] [sleep interval]
+#
+run_until_success() {
```

Comment:
I'm not sure how I would name this, but the current name makes me think it will just retry forever.

Reply:
Ha, good point. I will rename it to something more meaningful next time I'll be using it.